### PR TITLE
#48: DatePickerInput does not close if selecting same date (closes #48)

### LINKED ISSUE #48: DatePickerInput does not close if same date is inserted
steps to reproduce:
- insert a date as usual
- open date picker and re insert the same date again
- date picker will not disappear

### PR DESCRIPTION
**Pull request by FrancescoCioria**

### CODE REVIEW

**Comment by FrancescoCioria** [comment]
**test plan**: Tested it using DatePickerInput fields in the examples

File: test/tests/DatePickerInput-test.js [hunk 1]
```diff
@@ -138,5 +138,55 @@ describe('DatePickerInput', function() {
...
+  //   let datePickers = TestUtils.scryRenderedComponentsWithType(inputWrapper, DatePicker);
+  //   expect(datePickers.length).toBe(0);
+
+  //   const datePickerInputArea = TestUtils.findRenderedDOMComponentWithTag(inputWrapper, 'input');
+  //   TestUtils.Simulate.click(datePickerInputArea);
+
+  //   datePickers = TestUtils.scryRenderedComponentsWithType(inputWrapper, DatePicker);
+  //   expect(datePickers.length).toBe(1);
+
+  //   TestUtils.Simulate.keyUp(datePickerInputArea, {key: 'Enter'});
```
Comment (gabro) [inline test/tests/DatePickerInput-test.js]: Try: http://stackoverflow.com/questions/30628093/reacts-testutils-simulate-keydown-does-not-work

**Comment by ascariandrea** [comment]
@FrancescoCioria Shouldn't `package.json` be updated with the new npm version?

**Comment by FrancescoCioria** [comment]
@ascariandrea not yet, I've made a lot of changes (some of which breaking) since last version so we'll publish everything as v2.0.0
When the pending pull request #47 is merged I will publish v2.0.0

**Comment by ascariandrea** [comment]
@FrancescoCioria 
Understood, so I have to reference the commit-ish github url in my `package.json`.. Can we add at least a tag to this commit?

**Comment by gabro** [comment]
@ascariandrea #47 is quite small, and I already review it. I'd say @FrancescoCioria will be able to publish 2.0.0 very soon.

**Comment by ascariandrea** [comment]
@gabro 
Cool, cause I have a related issue in the lexdoit project due to this bug.

**Pull request merged**